<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="rawImageGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Raw Image</string>
    </property>
    <widget class="QLabel" name="rawImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>481</width>
       <height>391</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="transformedImageGroupBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Transform</string>
    </property>
    <widget class="QLabel" name="transformedImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>481</width>
       <height>391</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="avgTimeGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Average calc time [ns]</string>
    </property>
    <widget class="QLabel" name="avgTimeLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>496</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="noTransformRadioButton">
       <property name="text">
        <string>No Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gaussianBlurRadioBox">
       <property name="text">
        <string>Gaussian Blur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="flipRadioButton">
       <property name="text">
        <string>Flip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="laplacianRadioButton">
       <property name="text">
        <string>Laplacian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="motionTrackRadioButton">
       <property name="text">
        <string>Motion track</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
